eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -332,6 +332,7 @@ object NodeParams extends Logging {
       fundingBlockTarget = config.getInt("on-chain-fees.target-blocks.funding"),
       commitmentBlockTarget = config.getInt("on-chain-fees.target-blocks.commitment"),
       commitmentWithoutHtlcsBlockTarget = config.getInt("on-chain-fees.target-blocks.commitment-without-htlcs"),
+      disableCommitmentWithoutHtlcsFeeBumping = config.getBoolean("on-chain-fees.target-blocks.disable-commitment-without-htlcs-fee-bumping"),
```

Comment:
I would avoid double-negative, how about `on-chain-fees.target-blocks.spend-anchors-when-no-htlcs` or `on-chain-fees.target-blocks.always-spend-anchors`?

Reply:
Agreed, I really was in a bad variable-naming day :sweat_smile: 
Done in https://github.com/ACINQ/eclair/pull/2246/commits/e2ddd111b205347440b7cf9f7aeec5eb85d461a6